lib/device.rb

```diff
@@ -0,0 +1,18 @@
+class Device
+  attr :id
+
+  def initialize(identifier)
+    raise "Valid UDID or Serial Number required" unless is_serial?(identifier) || is_udid?(identifier)
+    id = identifier
```

Comment:
we need to store if it's a serial or UDID, and have a common method to pump out the identifier in a format that's either UDID or sn-!-SERIALNUMBER